chain-cli/src/commands/network-prune/index.ts

```diff
@@ -56,8 +60,11 @@ function downBrowserApi(fabloRoot: string): void {
   }
 }
 
-function getOrCreateFabloRoot(fabloDir: string): string {
-  const fabloRoot = path.resolve(fabloDir);
-  execSync(`mkdir -p "${fabloRoot}"`);
-  return fabloRoot;
+function getFabloRoot(fabloDir: string): string {
+  if (fs.existsSync(fabloDir)) {
+    return path.resolve(fabloDir);
+  } else {
+    console.log(`Directory '${fabloDir}' does not exist.`);
+    return "";
+  }
```

Comment:
This way maybe? To have it more explicit?

```suggestion
function getFabloRoot(fabloDir: string): string | undefined {
  if (fs.existsSync(fabloDir)) {
    return path.resolve(fabloDir);
  } else {
    console.warn(`Directory '${fabloDir}' does not exist.`);
    return undefined;
  }
```